src/redux-ducks/library.js

```diff
@@ -2,7 +2,7 @@
 import { Server } from "api";
 import type { LibraryFlagsType, LibraryFlagsPossibleValueTypes } from "types";
 import { ADD_MANGA } from "./mangaInfos";
-import { handleHTMLError, transformToMangaIdsArray } from "./utils";
+import { handleHTMLError, transformToMangaIdsArray } from "./utils"; import type {LibraryManga} from "@tachiweb/api-client";
```

Comment:
add new line

Reply:
Nice catch! I didn't look at all the eslint errors before I commited so I probably missed this.